common/defs/solana/programs.json

```diff
@@ -1692,6 +1692,10 @@
             {
               "account": "owner",
               "display_name": "Mint authority"
+            },
+            {
+              "account": "token_mint",
```

Comment:
Let's move this to the first place under `ui_properties` since the order also determines how the information gets displayed.

And then please change `display_name` of `amount` parameter to `Burn amount`.